backend/corpora/dataset_processing/process.py

```diff
@@ -242,30 +282,12 @@ def main():
     print(metadata_dict, flush=True)
     update_db(metadata=metadata_dict)
 
-    loom_filename = make_loom(local_filename)
-    cxg_dir = make_cxg(local_filename)
-    seurat_filename = make_seurat(local_filename)
-    artifacts = create_artifacts(local_filename, seurat_filename, loom_filename)
     bucket_prefix = join(os.environ.get("REMOTE_DEV_PREFIX", ""), os.environ["DATASET_ID"]).strip("/")
+    cxg_dir, loom_filename, seurat_filename, exceptions = convert_files_ignore_exceptions(local_filename)
```

Comment:
Just curious, how are we using the `exceptions` variable? Are we logging them or something 😄 ? Thank you!

Reply:
Good question -- We are already logging when the exception is caught so I didnt think it made sense to log again. It may be unnecessary but I thought it might be useful to have them all in one place for debugging. I can also just remove it.

Reply:
Ooh I see! So the `print` statement in each catch block is how we print errors to CloudWatch, right? Awesome!

Yeah I don't have a strong opinion on whether to keep the `exceptions` variable or not 😄👍

Thank you!

Reply:
is there a reason we aren't using the `logging` library?

Reply:
I was just following the pattern in the rest of the code. I thought @mckinsel may have had some reason for this approach but if not I can do an update through the whole file.

Reply:
it looks like @Bento007 is updating this in his pr to update conversion/validation status

Reply:
Yeah would be better to use logging